src/Microsoft.SqlTools.ServiceLayer/ObjectExplorer/SmoModel/SmoQueryModelDefinition.xml

```diff
@@ -35,14 +35,37 @@
 
   <Node Name="SqlErrorMessage" Type="UserDefinedMessage" Parent="Server" />
 
-  <Node Name="SqlTable" Parent="Database" />
+  <Node Name="SqlTable">
+    <Parent>Database</Parent>
+    <Parent>Schema</Parent>
+    <NavigationPath Parent="Database" Field="Tables" />
```

Comment:
Please update the schema definition at the top with the additional functionality you're adding here (and making it more clear in general wouldn't hurt either, right now it's a little light on details and seems to be incomplete.

Reply:
Ping @aasimkhan30